src/lib/api.js

```diff
@@ -0,0 +1,11 @@
+import { db } from './firebase';
+import { collection, addDoc } from 'firebase/firestore';
+
+const colRef = collection(db, 'Products');
```

Comment:
maybe this variable name `colRef` could be improved, not sure what the variable holds

Reply:
Personal opinion: I think that a possible name variable might be `productsRef` or `productsCollection`. But it's your decision, team. 😄 .